frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -125,19 +135,25 @@ export default function StatusChecker() {
       <Form method="post" noValidate>
         <input type="hidden" name="_csrf" value={csrfToken} />
         <div className="space-y-6">
-          <InputField id="sin" name="sin" label="Please enter your SIN" required />
-          <InputField id="code" name="code" label="Please enter your application code" required />
+          <InputField id="code" name="code" label={t('status:form.application-code-label')} helpMessagePrimary={t('status:form.application-code-description')} required />
+          <InputField id="sin" name="sin" label={t('status:form.sin-label')} required />
         </div>
         <Button className="mt-8" id="submit" variant="primary">
-          Check status
+          {t('status:form.submit')}
         </Button>
       </Form>
 
       {actionData && (
-        <dd className="mt-8">
-          <dt className="font-semibold">Status:</dt>
-          <dd>{actionData.status ?? 'No application'}</dd>
-        </dd>
+        <>
+          {clientStatusList.map((status) => (
+            <ContextualAlert type="info" key={status.id}>
+              <div>
+                <h3 className="mb-2 font-bold">Status</h3>
```

Comment:
Should this be an `<h2>`? 

Using an `<h3>` would mean that "Status" is a sub-heading under "Changes to these Terms of Use"